### PR TITLE
added stegcracker package

### PR DESCRIPTION
**Pull request by DerpGusta**
# Created a PKGBUILD for stegcracker

Stegcracker- A tool that brute forces the password for Steghide-a steganography tool,using wordlists

### CODE REVIEW

File: packages/stegcracker/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=stegcracker
+pkgver=2.0.7
+pkgrel=1
+pkgdesc='Steganography brute-force utility to uncover hidden data inside files'
```
Comment (Edu4rdSHL) [inline packages/stegcracker/PKGBUILD]: Add ending dot.

File: packages/stegcracker/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  cd "$pkgname-$pkgver"
+
+  python setup.py build
+}
+
+package() {
+  cd "$pkgname-$pkgver"
+
+  python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build
+}
```
Comment (Edu4rdSHL) [inline packages/stegcracker/PKGBUILD]: Add final newline.

File: packages/stegcracker/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=stegcracker
+pkgver=2.0.7
+pkgrel=1
+pkgdesc='Steganography brute-force utility to uncover hidden data inside files'
+arch=('any')
+groups=('blackarch' 'blackarch-stego')
+url="https://github.com/Paradoxis/StegCracker"
```
Comment (Edu4rdSHL) [inline packages/stegcracker/PKGBUILD]: Use single quotes.

File: packages/stegcracker/PKGBUILD [hunk 4]
```diff
@@ -4,14 +4,14 @@
...
+pkgdesc='Steganography brute-force utility to uncover hidden data inside files.'
 arch=('any')
 groups=('blackarch' 'blackarch-stego')
-url="https://github.com/Paradoxis/StegCracker"
+url='https://github.com/Paradoxis/StegCracker'
 license=('MIT')
 depends=('python')
 makedepends=('python-setuptools')
-source=("https://files.pythonhosted.org/packages/bd/9a/c5ee56f0b8d18c4664ae161e2f9b65e5fd0e4c5e5ab04c956386e2c8f5e2/$pkgname-$pkgver.tar.gz")
+source=('https://files.pythonhosted.org/packages/bd/9a/c5ee56f0b8d18c4664ae161e2f9b65e5fd0e4c5e5ab04c956386e2c8f5e2/$pkgname-$pkgver.tar.gz')
```
Comment (Edu4rdSHL) [inline packages/stegcracker/PKGBUILD]: Please don't change it, if you use single quotes here then the pkgver and pkgname variables aren't read. Rever the change in it line.

**Comment by Edu4rdSHL** [comment]
Thanks a lot, @DerpGusta.

**Comment by DerpGusta** [comment]
Thank you for bearing with me. It's my first time contributing to a big project haha

**Comment by Paradoxis** [comment]
Hi there! Awesome to see my tool included in the blackarch project :)
Just wanted to let you know the most recent version has been bumped to 2.0.9 as of today
Cheers!

**Pull request merged**